stock_inventory_exclude_sublocation/views/stock_inventory_view.xml

```diff
@@ -9,7 +10,7 @@
     <field name="model">stock.inventory</field>
     <field name="inherit_id" ref="stock.view_inventory_form"/>
     <field name="arch" type="xml">
-        <field name="filter" position="after">
+        <field name="location_id" position="after">
```

Comment:
Indeed, it's logical. To be complete, I would have put :
`groups="stock.group_stock_multi_locations"`

As location appear only in that case.

Reply:
Yeah, good idea. Done!